src/data/content/community/piano-attivita.yaml

```diff
@@ -268,44 +254,71 @@ components:
-          text: | #C
-            Inizio lavorazioni sulla sezione **Design pattern** del design system .italia. Approfondimenti dedicati a consolidare standard condivisi e aperti su come assemblare e utilizzare le risorse del design system per creare esperienze e contenuti in linea con le migliori pratiche e le norme per la progettazione.
+          title: "2024 — Q3" #C
+          headingLevel: 3
+          noSpace: false
 
         - name: TextImageCta
           headingLevel: 4
           specular: false
-          title: "Sito Designers Italia: miglioramento continuo" #C
+          title: "Design system .italia v1.0.0" #C
           text: | #C
-            Prosecuzione dei lavori di consolidamento del nuovo sito Designers Italia; in particolare avvio della seconda sessione di test di usabilità e lavorazioni per ottimizzare l'esperienza utente, la manutenibilità e la scalabilità nel tempo.
+            .italia, il design system del Paese, esce dalla fase beta con un censimento completo dell'accessibilità e dello stato dei componenti nelle risorse dell'ecosistema, e l'integrazione delle ultime funzionalità e contenuti nella documentazione. 
           ctas:
-          - label: Vai alla board di progetto per seguire i lavori #C
-            url: https://github.com/orgs/italia/projects/15 #M
+          - label: Vai alla board del progetto per seguire i lavori #C
+            url: https://github.com/orgs/italia/projects/17 #M
             blank: true #M #C true if new tab for external links
-            screenReaderText: " (si apre in una nuova finestra)"
             icon:
               icon: sprites.svg#it-github #C #I #it-arrow-right | it-external-link for external links
               color: primary
               align: middle
               hidden: true
               size: sm
               addonClasses: ms-2
-
-# ————— THE PAST ————— #
-
-    - title: I risultati che abbiamo raggiunto
-      headingLevel: 2
-      background: light
-      menu: false
-      centered: true
-      text: | #C
-        Le attività più importanti, gli obiettivi e i traguardi realizzati
-
-      components:
+          - label: Vai al changelog #C
+            url: "/design-system/fondamenti/versionamento/" #M
+            blank: false #M #C true if new tab for external links
+            icon:
+              icon: sprites.svg#it-arrow-right #C #I #it-arrow-right | it-external-link for external links
+              color: primary
+              align: middle
+              hidden: true
+              size: sm
+              addonClasses: ms-2
+        
+        - name: TextImageCta
+          headingLevel: 4
+          specular: false
+          title: "Nuovo kit di ricerca a supporto della fase 'Comprendere'" #C
+          text: | #C
+            Pubblicazione di attività e strumenti che consentono l'analisi dello stato dell'arte di siti e servizi digitali, a complemento di quelli dedicati al coinvolgimento diretto degli utenti.
```

Comment:
@zetareticoli @Giaimo-DTD forse questo testo, ora che è in chiava "cosa fatta" va rivisto. Così come la CTA che segue che ora porta al solo kit interviste.